modzy/models.py

```diff
@@ -454,47 +505,85 @@ def deploy(
         run_timeout_body = int(run_timeout)*1000 if run_timeout else 60000
         status_timeout_body = int(status_timeout)*1000 if status_timeout else 60000
 
-        model_metadata = {
-            "requirement": {"requirementId": -6 if gpu else 1},
-            "timeout": {
-                "run": run_timeout_body,
-                "status": status_timeout_body
-            },
-            "inputs": input_details or self.default_inputs,
-            "outputs": output_details or self.default_outputs,    
-            "statistics": performance_metrics or [],
-            "processing": {
-                "minimumParallelCapacity": 0,
-                "maximumParallelCapacity": 1
-            },
-            "longDescription": long_description or "",
-            "technicalDetails": technical_details or "",
-            "performanceSummary": performance_summary or ""
-        }
-        model_data = self._api_client.http.patch(f"{self._base_route}/{identifier}/versions/{version}", model_metadata)
-        self.logger.info(f"Model Data: {json.dumps(model_data)}")
+        if arch=="arm":
+            # assign "ARM" hardware requirement to bypass validation tests
+            model_metadata = {
+                "requirement": {"requirementId": -99},                
```

Comment:
You might want to consider making this a constant somewhere visible (like at the top of this file) and then using the constant here. Burying magic values in the middle of code is usually considered bad practice since it's darn near impossible to find later unless you happen to know what it is and where to look. At least with a constant, it describes what it is and you can put all the magic values in the same place.